src/XRP/default-xrp-client.ts

```diff
@@ -506,11 +544,11 @@ export default class DefaultXrpClient implements XrpClientDecorator {
     const accountData = await this.getAccountData(classicAddress.address)
     const openLedgerSequence = await this.getOpenLedgerSequence()
 
-    const senderAccountAddress = new AccountAddress()
-    senderAccountAddress.setAddress(wallet.getAddress())
+    const accountAddressToUnauthorize = new AccountAddress()
```

Comment:
All the changes here in `prepareBaseTransaction` seem like maybe a find and replace went too far??

Reply:
![Uploading sailor-moon-noooo.gif…]()

got it